### ISSUE TITLE
Wrong hash after updating to 3.3.6

### ISSUE THREAD

**Issue by endbay** [cascornelissen/svg-spritemap-webpack-plugin#82]
Hi,

`3.3.6` breaks something for me. 
```
        new SVGSpritemapPlugin(path.resolve(__dirname, 'assets/svg/*.svg'), {
            output: {
                filename: 'static/svg/sprite.[hash].svg',
                svg4everybody: false,
                svgo: {
                    removeEmptyAttrs: false,
                    removeUselessStrokeAndFill: false
                }
            },
            sprite: {
                prefix: false,
                generate: {
                    title: false,
                    use: true
                }
            }
        })
```

I'm using this configuration.. this was working till version `3.3.5`, but `3.3.6` break this. The Hash is not the right one anymore.

I'm loading the sprite svg with variable `__webpack_hash__`, but this hash does not match anymore in `3.3.6`.

**Comment by cascornelissen** [comment]
Looks like I've made an assumption in the `3.3.6` release that doesn't always hold true 🤔  Could you try the latest `master` release and report if that solves your issue?

```shell
npm install --save-dev cascornelissen/svg-spritemap-webpack-plugin#master
```

**Comment by endbay** [comment]
This is working, thanks!

**Comment by cascornelissen** [comment]
@endbay, thanks for the quick reply. I'm hoping this change fixes #83 as well, if it does I'll release a new patch version. I'll leave the issue open until a release is published ✌️

**Comment by cascornelissen** [comment]
@endbay, the fix for this issue has been released in version [`3.3.7`](https://github.com/cascornelissen/svg-spritemap-webpack-plugin/releases/tag/v3.3.7) 🚀
